ansys/dpf/post/dpf_path.py

```diff
@@ -66,35 +66,42 @@ def __init__(self, coordinates, scoping=None):
         ...     coord_copy[1] = coord_copy[0] + i * 0.001
         ...     coordinates.append(coord_copy)
         >>> dpf_path = post.DpfPath(coordinates=coordinates)
+
         """
-        self._coordinates = coordinates
+        # check inputs
         error_text = """coordinates attribute must
         be a list of list of int."""
         if not isinstance(coordinates, list):
```

Comment:
@anslpa can't we support the Field here?
More generally speaking, excessively checking types in python is not a good practice (python idiomatic is [“it’s easier to ask for forgiveness than permission”](https://devblogs.microsoft.com/python/idiomatic-python-eafp-versus-lbyl/).)
Here for example, you will raise if coordinates is an numpy array. You should not make any check here, the line `self._field.data = coordinates` will raise a meaningful exception if necessary and field.data is the only method that knows what it supports

Reply:
@cbellot000 I can add the possibility to define coordinates as a field, but I think this is needed to have the possibility to set coordinates as an array (as far as one of the goals of the ansys-dpf-post module is to have access to a post processing API thta does not require knowledges about ansys-dpf-core data model)